css/style.css

```diff
@@ -0,0 +1,7727 @@
+@charset "UTF-8";
```

Comment:
Could you make sure you run the compressed flag when compiling your sass. It looks like the styles are uncompressed which adds a lot to the file size.

Reply:
This should be fixed now.